common/test/test-theme/cms/static/sass/partials/_variables.scss

```diff
@@ -0,0 +1,255 @@
+// studio - utilities - variables
```

Comment:
We should consider adding theming tests for pattern library pages too, since we are starting to land a number of these. I'm okay with doing this in a follow-up PR.